_posts/en/newsletters/2024-03-13-newsletter.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Bitcoin Optech Newsletter #293'
+permalink: /en/newsletters/2024/03/13/
+name: 2024-03-13-newsletter
+slug: 2024-03-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Trustless onchain betting on potential soft forks:** ZmnSCPxj
+  [posted][zmnscpxj bet] to Delving Bitcoin a protocol for giving
+  control over a UTXO to a party that correctly predicts whether or not
+  a particular soft fork will activate.  For example, Alice thinks a
+  particular soft fork will activate and she's interested in acquiring
+  more bitcoins; Bob is also interested in acquiring more bitcoins but
+  does not think that fork will activate.  They agree to combine some
+  number of their bitcoins, at a ratio they both agree upon (e.g. 1:1),
+  so that Alice receives all of the combined bitcoins if the fork
+  activates by a certain time and Bob receives all of the combined
+  bitcoins if it does not.  If, before the deadline, a persistant
+  chainsplit occurs where one chain activates the fork and the other
+  forbids it, Alice receives the combined bitcoins on the activated
+  chain and Bob receives the combined bitcoins on the
+  activation-forbidden chain.
+
+    The basic idea has been proposed before ([example][rubin bet]) but
```

Comment:
Not sure if this is useful to point to, but Ill drop it here anyways:
https://bitcoinops.org/en/newsletters/2023/06/28/#speculatively-using-hoped-for-consensus-changes

Reply:
I think that's too different, so not including it, but thanks!